lib/hammer_cli_katello/host_bootc.rb

```diff
@@ -0,0 +1,34 @@
+module HammerCLIKatello
+  class HostBootc < HammerCLIKatello::Command
+    resource :host_bootc_images
+    desc "Manage bootc images on your hosts"
+
+    class ImagesCommand < HammerCLIKatello::ListCommand
+      action :bootc_images
+      command_name "images"
+
+      output do
+        field :bootc_booted_image, _("Bootc running image")
+        field :bootc_booted_digest, _("Bootc running image digest")
```

Comment:
```suggestion
        field :bootc_booted_image, _("Running image")
        field :bootc_booted_digest, _("Running image digest")
```
To align with https://github.com/Katello/hammer-cli-katello/commit/f96ac2110daa075285de69e4cf99b5c4486966e7#diff-46014eb7623d50b912d0ff35a3c09161a3d59ea36da6425c611b3582ce9b0de7